### DISCUSSION TITLE
ダークモードとライトモードの切り替えが誤爆しやすい

### DISCUSSION THREAD

**Discussion by anatawa12** [misskey-dev/misskey#14067] [General]
ダークモードとライトモードの切り替えが誤爆しやすいのでそれに関する議論をしたい

transfer from https://github.com/misskey-dev/misskey/pull/13512

**Comment by kakkokari-gtyih** [comment]
- [こういう外部ツール](https://forest.watch.impress.co.jp/docs/news/1442672.html) でなんとかしてもらうという前提でショートカットキー削除
- 1キーで切り替えられる系のショートカットを `Ctrl`, `Alt`, `Shift` などの制御キーとの同時押しに変更する
- ショートカットキーの設定画面を新設して有効 / 無効を全部いじれるようにする（→ https://github.com/misskey-dev/misskey/issues/14066 ）

のどれかが丸そう

**Reply by anatawa12** [reply]
外部ツール/OSのメリットかつデメリットとしてはほぼ全てのアプリが切り替わることだけど、Misskeyだけ切り替えたいって人はどのくらいいるんだろう

**Comment by anatawa12** [comment]
すでに出ている話として、

Q: 日常用途でショートカットがほしいほど切り替えるか?
A: 昼夜で切り替えている

Q: OSの機能を使えばいいのでは?
A: windowsにはない。

Q: 自動切替を実装するのは?
A: misskeyのout of scopeでは

というところまで出てる。

**Comment by anatawa12** [comment]
あとはショートカットとしては消して、slash commandsとかcmd/ctrl + kアクション検索とかとして実装する?

**Reply by kakkokari-gtyih** [reply]
ワンキーで切り替えれるのが誤爆のもとな気がするので`ctrl+`とか`shift+`とかにすればいいだけの可能性もある

**Comment by Sayamame-beans** [comment]
今でも誤爆の話は時々見かけるところではありますね…(何かしら対応したさはありそうです)

**Comment by 4oZEF** [comment]
#17221 経由でこのディスカッション認知したけど、2年前から言われてたんですねこれ。
既に言われてるけど、仮にショートカットキーとして用意するとしても何かとの同時押しにした方がよいと思います。

**Comment by anatawa12** [comment]
<img width="248" height="805" alt="image" src="https://github.com/user-attachments/assets/62dbe098-20b9-4c2b-944b-48c4b7504a52" />

今まで案に出してなかったのでこっちでも。
このサイドメニューに項目を追加できるようにするのはどうでしょう。

使う人は使う模様ですが多くの人には使われない機能なのですでにカスタマイズで変更可能になってるか所に移動するのが妥当だと考えられます

**Reply by 4oZEF** [reply]
`Ctrl+D`とか`Shift+D`だとどうしても何かしらのキーボードショートカットに被る可能性があるので、画面上のボタンとして実装されるならそれが一番丸い気がしますね。

**Comment by fruitriin** [comment]
ショートカットキーがある事自体はよくて、間違えて押した時に「何が起きたのかわからないこと」「戻し方がわからないこと」が結構困りポイントな気がして
Dキーを押した時に(「ライトモード/ダークモードに切り替えました　戻すにはDキー) みたいなバルーンチップとか出てくるっていうのはどうかな？

-------

追記
この話はDキーの暗黙知を持っている/持っていないで反応が分かれてるところを、押した瞬間に知識を持つ側に移動するUX
知ってても押し間違えた時に眩しいから場所を移動しよう　とは別のスコープ
